VideoLocker/src/main/java/org/edx/mobile/util/MediaConsentUtils.java

```diff
@@ -14,6 +16,8 @@
  */
 public class MediaConsentUtils {
 
+    @Inject Config config;
```

Comment:
I actually don't know how our DI thing really works. What's the right thing to do here?